templates/pages/category.html

```diff
@@ -44,6 +45,7 @@ <h1 class="page-heading">{{category.name}}</h1>
     <main class="page-content" id="product-listing-container">
         {{#if category.products}}
             {{> components/category/product-listing}}
+            {{{region name="category_page_content"}}}
```

Comment:
Is this the right location? It looks like it would appear below the list of products and would *not* appear if there are no products.